### PR TITLE
migration to Sequelize

### PR DESCRIPTION
**Pull request by miami78**
#### What does this PR do?
migrate to Sequelize ORM
#### Description of Task to be completed?
migrate from using SQL to Sequelize

### CODE REVIEW

File: src/Utils/errorHandler.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class ErrorHandler extends Error {
```
Comment (yegow) [inline src/Utils/errorHandler.ts]: Hey @miami78, great work. I know TS hasn't been smooth on you. Could we have this renamed to `ErrorResponse` instead? I feel like `ErrorHandler` is how you name an error-handling route.

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const path = require('path')
```
Comment (yegow) [inline .sequelizerc]: We should have this using ES6 module syntax. After all, we are using TS to be able to use the latest features of JS, right?

File: src/Utils/errorHandler.ts (hunk 1 above)
Comment (miami78) [reply src/Utils/errorHandler.ts]: @yegow  you are right,let me change that

File: .sequelizerc (hunk 2 above)
Comment (miami78) [reply .sequelizerc]: @yegow I agree

**Pull request closed without merging**